### DISCUSSION TITLE
error in devtool::install()

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#306] [Archive] [answered]
HI @soniamitchell 
My bad, here is the new discussion. I changed the name on the remote and gives me this...
<img width="832" alt="Screen Shot 2021-02-16 at 12 49 18" src="https://user-images.githubusercontent.com/77616349/108065196-8de41c00-7055-11eb-9c79-3dca011b5e17.png">

**Comment by soniamitchell** [comment, marked as answer]
Hi @mikatakahashi, could you remove the dependency on `base` in your `DESCRIPTION` and `MikaTakahashiDiversity-package.R` file, then run `devtools::document()` to rebuild the `NAMESPACE` file? base R functions are always available to R packages, so you don't need to define them as dependencies 👍

**Reply by mikatakahashi** [reply]
Yes! it worked. Thank you so much!
